diff --git a/node_modules/next/dist/esm/lib/metadata/metadata.js b/node_modules/next/dist/esm/lib/metadata/metadata.js
index 5fda081..edbbddc 100644
--- a/node_modules/next/dist/esm/lib/metadata/metadata.js
+++ b/node_modules/next/dist/esm/lib/metadata/metadata.js
@@ -93,6 +93,7 @@ export function createMetadataComponents({ tree, pathname, parsedQuery, metadata
         }
         return /*#__PURE__*/ _jsx("div", {
             hidden: true,
+            suppressHydrationWarning: true,
             children: /*#__PURE__*/ _jsx(MetadataBoundary, {
                 children: /*#__PURE__*/ _jsx(Suspense, {
                     name: "Next.Metadata",
diff --git a/node_modules/next/dist/lib/metadata/metadata.js b/node_modules/next/dist/lib/metadata/metadata.js
index 25d7c63..017d208 100644
--- a/node_modules/next/dist/lib/metadata/metadata.js
+++ b/node_modules/next/dist/lib/metadata/metadata.js
@@ -138,6 +138,7 @@ function createMetadataComponents({ tree, pathname, parsedQuery, metadataContext
         }
         return /*#__PURE__*/ (0, _jsxruntime.jsx)("div", {
             hidden: true,
+            suppressHydrationWarning: true,
             children: /*#__PURE__*/ (0, _jsxruntime.jsx)(_boundarycomponents.MetadataBoundary, {
                 children: /*#__PURE__*/ (0, _jsxruntime.jsx)(_react.Suspense, {
                     name: "Next.Metadata",
